### DISCUSSION TITLE
New UI Roust (main branch)

### DISCUSSION THREAD

**Discussion by cwchristerw** [ngoduykhanh/PowerDNS-Admin#1400] [General]
In my opinion new UI doesn't look good when using latest dockerimage I'm not sure if there is something missing in there but it looks bad.

I have been using PowerDNS Admin many years and this kind of UI change will make UX bad. Mainly my problem is that new UI will hide actions away from the user and in that way makes it difficult to use.

Also dislike new approach to use only icons in every action that is button. My problem is with Dashboard.

**Comment by metrax** [comment]
Which version are you using?
If you are using the latest version from the main branch, you will get the developement version. This can contain PRs that are still in development/testing and sometimes they need to be fixed. For example, actually we are missing a font set. This will be fixed soon. (maybe this is also the reason for you UI problem)

If you want to have a working instance, please set your docker container to v0.3.0.

**Comment by cwchristerw** [comment]
Probably I have always used latest tag because usually its latest release. :smile: I dont know easy way to downgrade to 0.3.0 because of database.

**Comment by cwchristerw** [comment]
How can I help with fixing this issue? :smile:

**Comment by AzorianMatt** [comment]
@Metrax FYI I corrected the missing font issue that followed the dependency upgrades from @AgentTNT.

@cwchristerw it's unfortunate you feel that way. Perhaps take a moment to appreciate the following;

1) Given the current view designs for many data record tables (lists with actions), space comes at a premium and must be conserved to better support more use-cases such as smaller screens.

2) This change will affect some users negatively while simultaneously being positive for others. Ergo, it's important to remember that your users aren't the only users.

Simply put, I disagree with your statement about icon only controls being bad for UX. I have used similar strategies for years in production with very important projects with complete success. Think about how many different applications are built with controls that use visual representations only. There are numerous examples of how this approach continues to work just fine even for the layman. Sure, it doesn't work well for every user, but this can't please every user either and would fail trying to do so. I imagine if I started stripping other data points from the lists to make more room, I would receive complaints for that too.

Please also make sure that you have pulled the absolute latest image as your description sounds like you're working from a particular build that was missing a key set of font files briefly. You would know it because the majority of the icons in the navigation would also disappear along with the icons on most buttons.

As @Metrax also said, the :latest version of the Docker image represents the current dev build which is unstable and subject to change unpredictability at any time. I will change this in the future to represent the latest stable release once the project overall, has a more extensive image strategy in place.

**Reply by cwchristerw** [reply]
That sounds good plan!

I just mostly disliked that buttons weren't even squares but weirdly shaped :smile:

Good job everyone :smile: 

It was improvement that this project will get new look :stuck_out_tongue_winking_eye:
